trace/src/main/org/jetbrains/lincheck/trace/TraceRecorderTracePoints.kt

```diff
@@ -53,7 +51,7 @@ sealed class TRTracePoint(
 
     val codeLocation: StackTraceElement get() = CodeLocations.stackTrace(codeLocationId)
 
-    abstract fun toText(verbose: Boolean): String
+    abstract fun toText(appendable: TRAppendable, verbose: Boolean)
```

Comment:
Could we make `verbose` a parameter of `DefaultTRTextAppendable`, instead of passing it to every `toText()` invocation?